python-avd/pyavd/j2filters/hide_passwords.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+
+def hide_passwords(value: list | dict | str | int | None, hide_passwords: bool = False) -> str:
+    """
+    hide_passwords replaces the input data by "<removed>" if the hide_passwords parameter is true
```

Comment:
Replaces the input data with "<removed>" if the hide_passwords parameter is true.

Reply:
according to the code, it replaces with `<removed>` string

Reply:
yes sorry i mean the doc string should be like
Replaces the input data with "<removed>" if the hide_passwords parameter is true.

Reply:
yes please use the same in double quote. Its getting removed while commenting

Reply:
```suggestion
    Replaces the input data with "<removed>" if the hide_passwords parameter is true. 
```